PaypalServerSdk.Standard/Authentication/OAuthTokenExtensions.cs

```diff
@@ -0,0 +1,22 @@
+// <copyright file="OAuthTokenExtensions.cs" company="APIMatic">
```

Comment:
It looks like the .NET SDK is the only one with this change. All it's doing is moving an class with an extension method into its own file (which is an improvement), so it shouldn't change anything functionally. Just wanted to call this out and make sure it wasn't a change we were expecting.

Reply:
Yeah wasn't expecting this, but looks like it's an update to how APImatic is updating the oauth functionality.